NEWS.md

```diff
@@ -11,6 +11,7 @@ for human readability.
   to 1D and 3D on `TreeMesh`.
 - Implementation of 1D Linearized Euler Equations.
 - New analysis callback for 2D `P4estMesh` to compute integrated quantities along a boundary surface, e.g., pressure lift and drag coefficients.
+- Optional tuple parameter for `GlmSpeedCallback` called `semi_indices` to specify for which semidiscretization we need to update the GLM speed.
```

Comment:
```suggestion
- Optional tuple parameter for `GlmSpeedCallback` called `semi_indices` to specify for which semidiscretization of a `SemidiscretizationCoupled` we need to update the GLM speed.
```
Otherwise, IMO it is not quite clear that this is a feature for coupled semis.